packages/manager/apps/hpc-vmware-managed-vcd/vitest.config.js

```diff
@@ -18,6 +18,9 @@ export default defineConfig({
         'src/App.tsx',
         'src/index.tsx',
         'src/tracking.constant.ts',
+        '404.tsx',
+        'src/**/*.spec.ts',
+        'src/**/*.spec.tsx',
```

Comment:
Why exclude all the spec files ?

Reply:
we don't test the tests files so they should not be included in the coverage report